src/Entity/DatasetSubmission.php

```diff
@@ -1586,9 +1586,9 @@ public function getMetadataContacts()
      *
      * @see self::getDatasetContacts()
      *
-     * @return string
+     * @return string|null
      */
-    public function getPointOfContactName() : string
+    public function getPointOfContactName() : ? string
```

Comment:
Since this can be null, you might want to add |default or otherwise handle the possible null cases in:
templates/DatasetReview/submit.html.twig
templates/DatasetSubmission/submit.html.twig

Reply:
Good catch! Fixed it

Reply:
this typehint is correct, but should we also make the template handle the null gracefully?